### DISCUSSION TITLE
How to use nuxt 3.8.0 verbatimModuleSyntax

### DISCUSSION THREAD

**Discussion by facexyy** [nuxt/nuxt#23974] [Questions]
### Describe the feature

![image](https://github.com/nuxt/nuxt/assets/148040903/bd30be1e-ab6b-473f-af77-0623e2afa843)

![image](https://github.com/nuxt/nuxt/assets/148040903/c6932c38-4f4d-4abe-840b-e215b5484333)

I am nuxt3.7.4 and upgraded to 3.8.0. Do I need to modify all the type imports in the project?

![image](https://github.com/nuxt/nuxt/assets/148040903/7bd7c41b-e51d-4289-87b7-c974f387afcc)

original
![image](https://github.com/nuxt/nuxt/assets/148040903/edd593aa-d11c-419b-9905-1697ebd12247)

Do I need to modify all the above type imports to

![image](https://github.com/nuxt/nuxt/assets/148040903/3f21a48c-f976-4849-b8e5-082a071e14bb)

Is this done?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by danielroe** [comment]
Yes, you do. Exactly.

**Comment by facexyy** [comment]
![image](https://github.com/nuxt/nuxt/assets/148040903/1d6ebe2a-918a-41aa-8f5d-313ed35a68c0)

Thanks Reply

Because there are too many changes

**Reply by danielroe** [reply]
You could write a quick regexp to do it in VSCode Search + Replace.

**Comment by facexyy** [comment]
The nice project is running normally

**Comment by AloisSeckar** [comment]
I went through this yesterday 😄  
https://github.com/AloisSeckar/ELRHistory/commit/c90cb41520d5dafe6fb83b7135fb25ef6890908a

It can be overriden, though it is advised not to do so (see https://nuxt.com/blog/v3-8#type-import-changes)
```
export default defineNuxtConfig({
  typescript: {
    tsConfig: {
      compilerOptions: {
        verbatimModuleSyntax: false
      }
    }
  }
})
```
